src/app/modules/backbone/components/sidebar/sidebar.component.ts

```diff
@@ -1,28 +1,39 @@
-import { Component, OnInit } from "@angular/core";
+import { ChangeDetectorRef, Component, OnDestroy, OnInit } from "@angular/core";
 
 import { AuthService } from "@core/authentication/authentication.service";
 
 import { NAVIGATION } from "@app/utils/constants/navigation";
 import { PricingTierService } from "@app/core/pricing/pricing-tier.service";
-import { Observable } from "rxjs";
+import { Subscription } from "rxjs";
 
 @Component({
   selector: "app-sidebar",
   templateUrl: "./sidebar.component.html",
   styleUrls: ["./sidebar.component.scss"]
 })
-export class SidebarComponent implements OnInit {
+export class SidebarComponent implements OnInit, OnDestroy {
     navigation = NAVIGATION;
     openLabel = "";
-    priceTierActive$: Observable<boolean> = new Observable();
+    priceTierActive = false;
+    private subscriptions: Subscription = new Subscription();
 
     constructor(
         private authService: AuthService,
         private pricingTierService: PricingTierService,
+        private changeDetectorRef: ChangeDetectorRef,
     ) {}
 
     ngOnInit(): void {
-        this.priceTierActive$ = this.pricingTierService.isDiscountActive$;
+        this.subscriptions.add(
+            this.pricingTierService.isDiscountActive$.subscribe(value => {
+                this.priceTierActive = value;
+                this.changeDetectorRef.detectChanges();
```

Comment:
Of all the things I tried, this was the only thing that worked. Everything else failed to have the sidebar (and toggle) detect that any changes had occured.

Reply:
Well done!